catalog/core/catalog-core-impl/filter-proxy/src/main/java/ddf/catalog/filter/proxy/adapter/GeotoolsFilterAdapterImpl.java

```diff
@@ -319,6 +325,27 @@ public Object visit(PropertyIsEqualTo filter, Object delegate) {
         String functionName = filterValues.functionName;
         List<Object> functionArgs = filterValues.functionArgs;
 
+        //Special case to handle relative temporal queries
+        if (literal instanceof String && RELATIVE_TEMPORAL_MATCHER.matcher((String) literal)
```

Comment:
Another option is to have it parsed out as a function instead of having to doing a regex. if you add a RELATIVE function to GeoToolsFunctionFactory you could use cql like `created = RELATIVE('P2Y3M2DT1H6M')`. That being said the escape logic looking for a RELATIVE function to call PropertyIsBetween will be pretty much the same the only thing your avoiding is the regex.